src/Storage/AxolotlStore/TSStorageManager+Calling.m

```diff
@@ -0,0 +1,26 @@
+//
+//  Copyright (c) 2017 Open Whisper Systems. All rights reserved.
+//
+
+#import "TSStorageManager+Calling.h"
+
```

Comment:
nit: I think we should use `NS_ASSUME_NONULL_BEGIN`/`END` in all objc files (.h too).

(crazy exceptions may exist, but I don't think this is one)